### ISSUE TITLE
Add emerging platforms process/flow

### ISSUE THREAD

**Issue by travier** [coreos/fedora-coreos-tracker#1569]
### Describe the enhancement

Add a lighter process to add a new supported platform for Fedora CoreOS.

The idea is that for lightly supported platforms we do the Ignition and Afterburn integration work and we stop there. Notably we do not produce disk images.

We then document a procedure to take one of our exiting image and tweak it to get a valid disk image for the new platform.

This would significantly reduce the overhead of adding new platform to Fedora CoreOS while keeping it reasonably low effort for users to setup FCOS on their favorite platform.

See PR for this process in https://github.com/coreos/fedora-coreos-tracker/pull/1562

### System details

N/A

### Additional information

Previous discussion in https://github.com/coreos/fedora-coreos-tracker/issues/719

**Comment by travier** [comment]
Platforms that are candidate for this process:
- Hetzner: https://github.com/coreos/fedora-coreos-tracker/issues/1324
- Scaleway: https://github.com/coreos/fedora-coreos-tracker/issues/866
- Proxmox: https://github.com/coreos/fedora-coreos-tracker/issues/736 (unsure)
- OVHcloud: https://github.com/coreos/fedora-coreos-tracker/issues/666
- Oracle Cloud: https://github.com/coreos/fedora-coreos-tracker/issues/414
- CloudStack: https://github.com/coreos/fedora-coreos-tracker/issues/716

Basically any platform already supported by cloud-init could be considered (https://github.com/canonical/cloud-init/tree/main/cloudinit/sources.).

**Comment by jlebon** [comment]
We discussed this in [today's community meeting](https://meetbot-raw.fedoraproject.org//fedora-meeting-1/2023-09-13/fedora_coreos_meeting.2023-09-13-16.30.txt):

> AGREED: We generally like the idea of "emerging" platforms. We will continue fleshing out the UX and communications part of it in the ticket.  (jlebon, 17:05:29)

**Comment by travier** [comment]
We now have this flow as of https://github.com/coreos/fedora-coreos-tracker/pull/1562. We'll keep iterating and I'll close this one as it does not need immediate action.
